src/app/config/sgid.ts

```diff
@@ -0,0 +1,63 @@
+import { Schema } from 'convict'
+
+import { ISgidVarsSchema } from '../../types'
+
+const HOUR_IN_MILLIS = 1000 * 60 * 60
+const DAY_IN_MILLIS = 24 * HOUR_IN_MILLIS
+
+export const sgidVarsSchema: Schema<ISgidVarsSchema> = {
+  endpoint: {
+    doc: 'The sgID OIDC endpoint',
+    format: '*',
+    default: '',
+    env: 'SGID_ENDPOINT',
+  },
+  clientId: {
+    doc: 'The client id registered with sgID',
+    format: '*',
+    default: '',
+    env: 'SGID_CLIENT_ID',
+  },
+  clientSecret: {
+    doc: 'The client secret registered with sgID',
+    format: '*',
+    default: '',
+    env: 'SGID_CLIENT_SECRET',
+  },
+  privateKey: {
+    doc: 'The path to the private key to decrypt payloads from sgID. Also used for JWT signing',
+    format: '*',
+    default: '',
+    env: 'SGID_PRIVATE_KEY',
+  },
+  publicKey: {
+    doc: 'The path to the public key given to sgID. Also used to verify JWTs created by FormSG',
+    format: '*',
```

Comment:
should the format be `String` with a `null` default?

should we also rename this to `publicKeyPath` for clarity?